workflows/flux/flux_local.yaml

```diff
@@ -0,0 +1,71 @@
+description:
```

Comment:
Do we want to add this file to `merlin/examples/workflows/` instead? That's where the other flux workflows currently are.